conf/graphite/storage-schemas.conf

```diff
@@ -7,10 +7,7 @@
 
 # Carbon's internal metrics. This entry should match what is specified in
 # CARBON_METRIC_PREFIX and CARBON_METRIC_INTERVAL settings
-[carbon]
-pattern = ^carbon\.
-retentions = 60:90d
```

Comment:
I agree with that, but lets comment these lines rather than delete them. I'd like to keep them for reference to help folks that may want to customize.